conf/include/webkit-bleeding.inc

```diff
@@ -0,0 +1,14 @@
+# Configure select recipes to use the latest upstream source.
+# Add the following to local.conf to enable it:
+#
+#   require conf/include/webkit-bleeding.inc
+
+PREFERRED_VERSION_cog = "0.4.0+gitAUTOINC+%"
+PREFERRED_VERSION_libwpe = "1.4.0+gitAUTOINC+%"
+PREFERRED_VERSION_wpebackend-fdo = "1.4.0+gitAUTOINC+%"
+PREFERRED_VERSION_wpebackend-rdk = "1.20180925+gitAUTOINC+%"
+
+SRCREV_class-devupstream_pn-cog = "${AUTOREV}"
+SRCREV_class-devupstream_pn-libwpe = "${AUTOREV}"
+SRCREV_class-devupstream_pn-wpebackend-fdo = "${AUTOREV}"
+SRCREV_class-devupstream_pn-wpebackend-rdk = "${AUTOREV}"
```

Comment:
Would it make sense to do this for the wpewebkit recipe as well?

Reply:
Sure. We could make wpewebkit pull from the WebKit git mirror (git://git.webkit.org/WebKit-https.git) and then use `devupstream.bbclass` and `${AUTOREV}` just like here. Would you like me to do that?

Reply:
If it's not too much trouble, yes please. I think it makes sense in the context of this PR.